htdocs/product/reassort.php

```diff
@@ -141,7 +141,7 @@
 $sql .= $hookmanager->resPrint;
 $sql .= ' FROM '.MAIN_DB_PREFIX.'product as p';
 $sql .= ' LEFT JOIN '.MAIN_DB_PREFIX.'product_stock as s ON p.rowid = s.fk_product';
-$sql .= ' LEFT JOIN '.MAIN_DB_PREFIX.'entrepot as e ON s.fk_entrepot = e.rowid AND e.entity IN ('.getEntity('entrepot').')';
+$sql .= ' RIGHT JOIN '.MAIN_DB_PREFIX.'entrepot as e ON s.fk_entrepot = e.rowid AND e.entity IN ('.getEntity('stock').')';
```

Comment:
Why a RIGHT JOIN. We already have a LEFT JOIN so we take all product whateever is link into product_stock, so adding a RIGHT JOIN here will means we also have another master table. We can't have 2 master table in same SQL as this lead to unpredicable results.

Reply:
@eldy INNER JOIN ? or move `e.entity IN ('.getEntity('stock').')` after the WHERE ?

Reply:
@eldy i replace with INNER JOIN

Reply:
If you use a inner join, if the product has not stock, the line will not be returned.

Reply:
@eldy "reassort" with no stock (entrepot)  it's possible ? :-)